data/core/macros/abilities.cfg

```diff
@@ -388,6 +388,21 @@ Enemy units cannot see this unit while it is in deep water, except if they have
     [/dummy]
 #enddef
 
+#define ABILITY_EXPLODE TYPE VALUE
+    # Canned definition of the Explode ability to be included in an
+    # [abilities] clause.
+    # This works because of data/lua/explode_ability.lua
+    [dummy]
```

Comment:
I think it might be a good idea to use a different tag name, especially since it has some custom contents (`value` and `apply_to`).

Reply:
That's part of how to fix the schema complaints, I guess?

Reply:
Indeed, having a unique tag name certainly helps with making it validate in the schema.